DEV-3/DEV-3/DEV-3/FirstCriterion.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DEV_3
+{
+    /// <summary>
+    /// This is the first criterion.Condition: maximum productivity within the amount.
+    /// </summary>
+    class FirstCriterion : Recruitment
+    {   
+        public FirstCriterion(int Amount)
+        {
+            if (Amount > 0)
+            {
+                this.Amount = Amount;
+            }
+            else
+            {
+                throw new Exception("You don't have money. ");
+            }
+        }
+        /// <summary>
+        /// This method counts the maximum productivity within the amount.
+        /// </summary>
+        /// <returns>List of number of employees to order.</returns>
+        public override List<int> Choose()
+        {
+            while (Amount > 0)
+            {
+                if (objLead.Salary < Amount)
+                {
+                    ListOfEmployee[0]++;
```

Comment:
ListOfEmployee[0]
ListOfEmployee[1]
ListOfEmployee[2]
in this case it is a magic number

Is it better to use enum?